docs/topics/proc_installing-launcher-tool-addon.adoc

```diff
@@ -0,0 +1,63 @@
+// This is a parameterized module. Parameters used:
+//
+//   parameter-custom-catalog: A custom catalog is used instead of the default one
+//
+// Rationale: This module is useful both when setting up minishift as is and when testing a local
+// booster catalog.
+
+[id='installing-launcher-tool-addon_{context}']
+= Installing {launcher} Tool as an Add-on
+link:https://docs.openshift.org/latest/minishift/using/addons.html[Add-ons] enable you to extend the behavior of {OpenShiftLocal}, and the {launcher} tool link:https://github.com/minishift/minishift-addons/tree/master/add-ons/fabric8-launcher[provides an add-on] as an installation option.
+
+.Prerequisites
+* xref:starting-and-configuring-the-openshiftlocal-for-the-launcher-tool_{context}[{OpenShiftLocal} running].
+* A link:{link-launcher-openshift-local-install-guide}#creating-a-github-personal-access-token[GitHub personal access token].
+
+.Procedure
+
+. Download and install the {launcher} add-on.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ git clone git@github.com:minishift/minishift-addons.git
+$ minishift addon install minishift-addons/add-ons/fabric8-launcher/
+$ minishift addons apply fabric8-launcher \ 
+      --addon-env GITHUB_USERNAME=GH_USERNAME --addon-env GITHUB_TOKEN=TOKEN <1>
+----
+<1> Use your GitHub username and personal access token
+
+. Monitor the status of the {launcher} tool until it completes start up.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc project rhoarpad
```

Comment:
Might want to explain that the addon creates a new project, defaulting to the name `rhoarpad`.

(Which, frankly, is silly, but whatevz...)

Reply:
@lthon I think that information is evident form the note on line #29. WDYT?

Reply:
It is now. It wasn't when I added the comment :-)

(In other words: this comment has been addressed.)

Reply:
Nice:)  thank you:)